src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractImportCompletionProvider.cs

```diff
@@ -196,7 +196,8 @@ private static async Task<bool> IsInImportsDirectiveAsync(Document document, int
             var syntaxFacts = document.GetRequiredLanguageService<ISyntaxFactsService>();
             var syntaxTree = await document.GetRequiredSyntaxTreeAsync(cancellationToken).ConfigureAwait(false);
             var leftToken = syntaxTree.FindTokenOnLeftOfPosition(position, cancellationToken, includeDirectives: true);
-            return leftToken.GetAncestor(syntaxFacts.IsUsingOrExternOrImport) != null;
+            return leftToken.GetAncestor(syntaxFacts.IsUsingOrExternOrImport) is { } node
+                && !syntaxFacts.IsTerminator(node, leftToken);
```

Comment:
consider docing

Reply:
I added a bunch of docs to `IsTerminator`. Is that not clear?

Reply:
➡️ Covered by https://github.com/dotnet/roslyn/pull/44402#discussion_r428955803